config.py

```diff
@@ -83,6 +83,9 @@ class Configuration(BaseSettings):
     )
 
     bs_pop_endpoint: str = ""
+    pop_score_cutoff = 300
+    pop_score_exponent = 2 / 3
+    pop_score_range = 300
```

Comment:
```suggestion
    pop_score_range = 100
```

maybe we should make this 100, so it's nice and round?